blackbox/docs/config/license.rst

```diff
@@ -0,0 +1,16 @@
+.. _conf-license:
+
+================
+License Settings
+================
+
+The section documents settings that only apply to the CrateDB Enterprise License.
```

Comment:
"this section"

Reply:
copy-paste, the root of all evil. Should I change it in the session doc as well?